### DISCUSSION TITLE
UniDock - a new Avalonia based framework for docking

### DISCUSSION THREAD

**Discussion by npolyak** [AvaloniaUI/Avalonia#6493] [General]
I published UniDock - a new Avalonia based framework for docking on Github: https://github.com/npolyak/NP.Avalonia.UniDock and also an introductory article that describes it - https://www.codeproject.com/Articles/5311587/UniDock-new-Multiplatform-Docking-Framework-Introd. Please, try using the framework and send me your wishes for improvements. Thanks

**Comment by kekekeks** [comment]
https://github.com/AvaloniaCommunity/awesome-avalonia

**Reply by kekekeks** [reply]
Also note that "drag window over some point" approach won't work once we get a native Wayland backend.

**Reply by maxkatz6** [reply]
Won't wayland be optional? I thought it's not supported everywhere yet.

**Reply by npolyak** [reply]
we'll cross wayland bridge when we get to it. I am sure there will be 1000 of ways to do things in or around wayland.

**Reply by kekekeks** [reply]
@maxkatz6 by the time we get our backend working it will be almost everywhere.
